autoscaler/capacity.py

```diff
@@ -26,8 +26,13 @@
 
 def is_possible(pod):
     """
+    returns False if the pod does not require GPUs. Otherwise,
     returns whether the pod is possible under the maximum allowable capacity
     """
+
+    if pod.resources.get('alpha.kubernetes.io/nvidia-gpu') is None:
```

Comment:
Can you add a TODO to clean this up?

I feel like this is a bit of a hack, since we'd really want a filter applied higher-level in the process - say, filtering out pods immediately after listing them, instead of putting a conditional inside of an existing filter function.